src/library/scala/concurrent/Future.scala

```diff
@@ -20,11 +20,16 @@ import scala.collection.generic.CanBuildFrom
 import scala.reflect.ClassTag
 
 
-/** The trait that represents futures.
+/** 
  *
- *  Asynchronous computations that yield futures are created with the `Future.apply` call:
+ *  A `Future` represents a value which may or may not *currently* be available,
```

Comment:
I'd suggest starting to adopt the style here: http://docs.scala-lang.org/style/scaladoc.html

```
/** Hi there
  * 
  * Line 1
  * Line 2
  */
```

This has the advantage of surviving the auto formatting attentions of IDEA 16 code format and requires less column-edit-mode-fu to maintain.